### PR TITLE
[JBPM-9743] Event Emitters - Adding event context information

### PR DESCRIPTION
**Pull request by fjtirado**
Adding operation and initiator to TaskInstanceView

**JIRA**: 
[link](https://issues.redhat.com/browse/JBPM-9743)

**referenced Pull Requests**:
https://github.com/kiegroup/jbpm/pull/1932

### CODE REVIEW

**Comment by fjtirado** [comment]
jenkins run fdb

File: kie-internal/src/main/java/org/kie/internal/task/api/TaskPersistenceContext.java [hunk 1]
```diff
@@ -32,14 +33,23 @@
...
+    
+    default Task updateTask (Task task) {
+        return updateTask (task, Operation.Update);
+    }
+    
+    default Task updateTask (Task task, Operation operation) {
+        return updateTask (task, operation, Optional.empty());
+    }
+
+    Task updateTask (Task task, Operation operation, Optional<String> targetEntity);
```
Comment (gmunozfe) [inline kie-internal/src/main/java/org/kie/internal/task/api/TaskPersistenceContext.java:52]: As an internal API, doesn't need to be defined as default, does it?

**Review by gmunozfe — APPROVED** [review]
Looks good to me, well done @fjtirado !
Just a suggestion for one of the new actions, and a question regarding new method in the API

File: kie-internal/src/main/java/org/kie/internal/task/api/TaskPersistenceContext.java (hunk 1 above)
Comment (fjtirado) [reply kie-internal/src/main/java/org/kie/internal/task/api/TaskPersistenceContext.java:52]: Im not using default here to achieve backward compatibility, but to provide default implementations that I feel will be valid by any implementation

File: kie-internal/src/main/java/org/kie/internal/task/api/model/Operation.java [hunk 2]
```diff
@@ -34,5 +34,10 @@
     Exit,
     Nominate,
     Modify,
-    View;
+    View,
+    Create,
+    Archive,
+    Update,
+    Comment,
+    Attachment;
```
[the remark this answers is not in the document]
Comment (fjtirado) [reply kie-internal/src/main/java/org/kie/internal/task/api/model/Operation.java:42]: yes, you are right, changing

**Review by gmunozfe — APPROVED** [review]
Looks good to me, very well-done @fjtirado !

**Review by afalhambra — APPROVED** [review]
Great job

**Comment by fjtirado** [comment]
@gmunozfe @afalhambra Approach changed  as per discussion with customer, please re review

**Comment by fjtirado** [comment]
jenkins run cdb

**Comment by gmunozfe** [comment]
jenkins run fdb

**Pull request closed without merging**